gossip/src/crds_value.rs

```diff
@@ -44,8 +44,8 @@ pub const MAX_EPOCH_SLOTS: EpochSlotsIndex = 255;
 #[cfg_attr(feature = "frozen-abi", derive(AbiExample))]
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
 pub struct CrdsValue {
-    pub signature: Signature,
-    pub data: CrdsData,
+    signature: Signature,
+    data: CrdsData,
```

Comment:
this is going to cause breaking changes for gossip crate users. any concerns about this?

Reply:
I think leaving these fields as public was an oversight and they shouldn't have been exposed to begin with.
I doubt that anyone relies on these implementation details, and even if they are, they can easily update their code or make a fork of the crate.

Reply:
ya should be easy enough to update